lib/diagnostic.rb

```diff
@@ -71,18 +71,18 @@
 
 # What value will be returned. Write your response as a Ruby string.
 
-Diagnostic.flow_control = 'your response here'
+Diagnostic.flow_control = 'Just your average billionaire'
 
```

Comment:
Here it would actually be "The Dark Knight" 
The if statement checks if `batman` is true. Since we defined it above it is true and that block will execute